templates/netlify/remix.init/index.js

```diff
@@ -0,0 +1,102 @@
+const inquirer = require("inquirer");
+const fs = require("fs/promises");
+const { join } = require("path");
+const PackageJson = require("@npmcli/package-json");
+
+const filesToCopy = [
+  ["README.md"],
+  ["netlify.toml"],
+  ["edge-server.js", "server.js"],
+  ["remix.config-edge.js", "remix.config.js"],
+  ["vscode.json", join(".vscode", "settings.json")],
+  ["globals.d.ts", "globals.d.ts"],
+];
+
+const filesToModify = ["app/entry.server.tsx", "app/root.tsx"];
+
+async function modifyFilesForEdge(files, rootDirectory) {
+  const filePaths = files.map((file) => join(rootDirectory, file));
+  const contents = await Promise.all(
+    filePaths.map((path) => fs.readFile(path, "utf8"))
+  );
+
+  await Promise.all(
+    contents.map((content, index) => {
+      const newContent = content.replaceAll(
+        "@remix-run/node",
+        "@remix-run/deno"
+      );
+      return fs.writeFile(filePaths[index], newContent);
+    })
+  );
+}
+
+async function copyEdgeTemplateFiles(files, rootDirectory) {
+  for (const [file, target] of files) {
+    await fs.copyFile(
+      join(rootDirectory, "remix.init", file),
+      join(rootDirectory, target || file)
+    );
+  }
+}
+
+async function updatePackageJsonForEdge(directory) {
+  const packageJson = await PackageJson.load(directory);
+  const { dependencies, scripts, ...restOfPackageJson } = packageJson.content;
+
+  // Replaces remix dev since there is no remix dev server for custom servers
+  scripts["dev"] = "cross-env NODE_ENV=production netlify dev";
+
+  // The start script is not required as the dev script does the same thing
+  delete dependencies["start"];
+
+  delete dependencies["@remix-run/netlify"];
+  delete dependencies["@remix-run/node"];
```

Comment:
What about destructuring `dependencies` instead, so we're not mutating the original?

```suggestion
```